ttim/well.py

```diff
@@ -406,13 +464,19 @@ def __init__(
         self.nunknowns = self.nparam
 
     def initialize(self):
-        WellBase.initialize(self)
-        self.parameters = np.zeros(
-            (self.model.ngvbc, self.nparam, self.model.npval), "D"
-        )
+        super().initialize()
+        self.parameters = {}
+        for t_int in self.model.logtintervals:
+            self.initialize_interval(t_int)
         # Needed in solving for a unit head
         self.pc = self.aq.T[self.layers]
 
+    def initialize_interval(self, t_int):
+        super().initialize_interval(t_int)
+        self.parameters[t_int] = np.zeros(
+            (self.model.ngvbc, self.nparam, self.model.nppar), "D"
```

Comment:
dtype=complex

Reply:
This is now fixed. What is still undesirable is that now all Well classes have an `initialize_interval` function. Seems repetitive. Let's discuss.